modules/components/src/DataTable/DataTable.js

```diff
@@ -88,8 +103,8 @@ class DataTableWithToolbar extends React.Component {
       downloadUrl = urlJoin(ARRANGER_API, projectId, 'download'),
       onSortedChange = () => {},
       alwaysSorted = [],
-      initalSelectedTableRows = [],
-      keepSelectedOnPageChange = false,
+      initalSelectedTableRows,
+      keepSelectedOnPageChange = false, // If false, this will reset the selection to empty on reload even if sessionStorage is enabled. To keep selections after reload st this to true.
```

Comment:
```suggestion
      keepSelectedOnPageChange = false, // If false, this will reset the selection to empty on reload even if sessionStorage is enabled. To keep selections after reload set this to true.
```